### PR TITLE
log the version and build date on server startup

### PR DESCRIPTION
**Pull request by jboyd01**
for serviceability, log the version and build date on API Server and Controller Manager startup.  Results in log entries like this:
```Preparing to run Service Catalog API server v0.1.8-4+5317111b4e8932-dirty (built 2018-02-19T16:40:13Z)```
and
```Preparing to run Service Catalog Controller v0.1.8-4+5317111b4e8932-dirty (built 2018-02-19T16:40:13Z)```

### CODE REVIEW

File: cmd/apiserver/app/server/run_server.go [hunk 1]
```diff
@@ -56,7 +57,7 @@ func RunServer(opts *ServiceCatalogServerOptions, stopCh <-chan struct{}) error
 
 func runEtcdServer(opts *ServiceCatalogServerOptions, stopCh <-chan struct{}) error {
 	etcdOpts := opts.EtcdOptions
-	glog.V(4).Infoln("Preparing to run API server")
+	glog.Infof("Preparing to run Service Catalog API server %s (built %s)", version.Get().String(), version.Get().BuildDate)
```
Comment (MHBauer) [inline cmd/apiserver/app/server/run_server.go]: can we put this in the caller of runEtcdServer? I do not want there to end up duplicating code when we get a CRD server.

**Comment by MHBauer** [comment]
Actually, I imagine there is one single place we can put this in the hypercube caller. Can we eliminate the two copies and put it in the caller of both?

**Comment by jboyd01** [comment]
@MHBauer I looked at moving the logging up higher in the call chain that is common to both API Server and the Controller.  Logs are initialized in pkg.hyperkube which is copied out of K8s so I didn't want to modify that common code (nor do I feel it belongs there) or do the logging prior to initialization.  hyperkube invokes the RunServer() methods which is specific to API and Controller.

On the APIServer side I can move the logging up a level to the RunServer() making it independent of etcd/crd.

**Comment by jboyd01** [comment]
@MHBauer I was hesitant to do it in the hyperkube copied code as it makes updating from k8s (arguably) 
 (slightly)  more complex.  I've moved the version logging into hyperkube and validated our hyperkube is still pretty much in sync with k8s latest.

**Pull request merged**